src/router/index.js

```diff
@@ -1,8 +1,14 @@
 import { createRouter, createWebHistory } from 'vue-router';
+import Home from '../pages/Home.vue';
 import Register from '../pages/Register.vue';
 import Login from '../pages/Login.vue';
 
 const routes = [
+  {
+    path: '/',
+    name: 'Home',
+    component: Home,
```

Comment:
It's better that you will use lazy loading when it comes to page components (like `Home`).
Read more about the concept of lazy loading here: https://vueschool.io/lessons/how-to-lazy-load-routes-with-vue-router